app/views/users/_comments_section.html.erb

```diff
@@ -1,46 +1,65 @@
 <% cache("user-profile-comments-#{@user.last_comment_at}-#{@user.id}-#{@comments.size}", expires_in: 2.days) do %>
-  <style>
-    .index-comments header {
-      color: <%= HexComparer.new([user_colors(@user)[:bg], user_colors(@user)[:text]]).brightness(0.88) %>;
-    }
-    .night-theme .index-comments header {
-      color: #cedae2;
-    }
-  </style>
-  <div class="index-comments crayons-card">
+  <div class="profile-comment-card crayons-card mt-2 m:mt-0 mb-2">
     <% if @comments.present? %>
       <% if params[:view] == "comments" %>
-        <div class="breadcrumb"><a href="<%= @user.path %>">View All Activity</a></div>
-        <header>
-          <% if @user.comments_count > 250 %>
-            <h4>Last 250 Comments</h4>
-          <% else %>
-            <h4>All <%= @user.comments_count %> Comments</h4>
-          <% end %>
-        </header>
+        <div class="crayons-card__header">
+          <h4 class="crayons-card__headline">
+            <% if @user.comments_count > 250 %>
+              Last 250 comments
+            <% else %>
+              All <%= @user.comments_count %> comments
+            <% end %>
+          </h4>
+          <div class="crayons-card__actions">
+            <a href="<%= @user.path %>" class="crayons-link--branded fs-base">
+              View all activity
+            </a>
+          </div>
+        </div>
       <% else %>
-        <header>
-          <h4>Recent Comments</h4>
-        </header>
+        <div class="crayons-card__header">
+          <h4 class="crayons-card__headline">
+            Recent comments
+          </h4>
+        </div>
       <% end %>
     <% end %>
     <% @comments.each do |comment| %>
       <% if comment.commentable.present? && comment.commentable.published && !comment.deleted %>
-        <a href="<%= comment.path %>">
-          <div class="single-comment <%= "strong-comment" if comment.ancestry.nil? %>">
-            <span class="comment-title">re: <%= comment.commentable.title %></span>
-            <span class="comment-date">
+        <a href="<%= comment.path %>" class="profile-comment-row ">
+          <span class="fw-bold fs-base">
+            <%= comment.commentable.title %>
+          </span>
+          <div class="inline-flex items-center gap-2">
+            <div class="fs-s comment-preview">
+              <div class="crayons-avatar crayons-avatar--m mr-1">
+                <img alt="<%= @user.username %>%" src="<%= @user.profile_image_90 %>" class="crayons-avatar__image" />
+              </div>
+              <span class="fw-medium mr-1">
+                <%= @user.name %>
+              </span>
```

Comment:
I'm not sure if we need to show the avatar and username here.. This section is on user profile anyway, so it's pretty obvious who's the author of the comment. 

I do know we show it in the post cards below but it's because it's the way it work ¯\_(ツ)_/¯ . I think intentional inconsistency could be pretty beneficial here :)